inject/chrome.js

```diff
@@ -13,6 +13,7 @@ const allowedMessages = [
     'getClickToLoadState',
     'getYouTubeVideoDetails',
     'openShareFeedbackPage',
+    'addDebugFlag',
```

Comment:
Is this still needed, aren't you using the other transport?

Reply:
This is needed for MV2. We haven't implemented a proper messaging transport for the extension.

Reply:
Ok thanks for the heads up :)